tensorflow/core/api_def/base_api/api_def_LeakyRelu.pbtxt

```diff
@@ -0,0 +1,4 @@
+op {
+  graph_op_name: "LeakyRelu"
+  summary: "Computes rectified linear: `max(features, features * alpha)`."
+}
```

Comment:
Can you add visibility:hidden here?

I think this will obviate the need for the API golden file changes since there's already tf.nn.leaky_relu and no need for tf.leaky_relu (which gets implicitly created when an op is added)